tests/lax_numpy_test.py

```diff
@@ -2565,6 +2565,12 @@ def testArrayUnsupportedDtypeError(self):
                                 "JAX only supports number and bool dtypes.*"):
       jnp.array(3, [('a','<i4'),('b','<i4')])
 
+  def testArraySupportsFloat0(self):
+    with self.assertRaisesRegex(TypeError,
+                                "JAX only supports number and bool dtypes.*"):
+      jnp.array(3, jax.float0)
+      jnp.zeros([], jax.float0)
```

Comment:
Should you be testing if `jnp.zeros([], jax.float0)` is supported instead of throwing an error? Do we want to support `jnp.array(3, jax.float0)`?

Reply:
You are totally right, have fixed.